scss/titles.scss

```diff
@@ -2,21 +2,21 @@ $title-decoration-height: 90px;
 $title-decoration-width: 100px;
 
 .title {
-	@extend %roboto;
-	block-size: $text-big;
+  @extend %roboto;
```

Comment:
No need for this extend since now it will automatically use the default value in the equalizer